funcr/funcr.go

```diff
@@ -740,7 +744,16 @@ func (f Formatter) FormatInfo(level int, msg string, kvList []any) (prefix, args
 	if policy := f.opts.LogCaller; policy == All || policy == Info {
 		args = append(args, "caller", f.caller())
 	}
-	args = append(args, "level", level, "msg", msg)
+	if f.opts.LogInfoLevel == nil {
```

Comment:
Why not:

```
if f.opts.LogInfoLevel == nil {
	args = append(args, "level", level)
} else {
	lvKey := *f.opts.LogInfoLevel
	if lvKey != "" {
		args = append(args, lvKey, level)
	}
}
args = append(args, "msg", msg)
```

Does that have some significant impact on benchmark?

Reply:
Even simpler - look at `newFormatter`.  It initializes some fields.  Just do this nil check one time there, like:

```
if opts.LogInfoLevel == nil {
    key := "level"
    opts.LogInfoLevel = &key
}
```
... and then this becomes just

```
if key := opts.LogInfoLevel; key != "" {
    args = append(args, key, level)
}
args = append(args, "msg", msg)
```